src/resources/postcss/my-tickets/_orders.pcss

```diff
@@ -86,22 +87,23 @@
 
 				& > div {
 					flex: 1 0 100%;
+					flex-basis: var(--tec-grid-width-1-of-4);
```

Comment:
I added this piece because the content wasn't breaking properly when resizing the page. However, I'm a bit on the fence about locking the "My Tickets" page into always having four columns. As it stands, the page shifts between two, three, or four columns, but setting it to four by default might throw things off, especially if ET+ or ETWP aren't being used, or if down the line we decide to switch up the column count. Does anyone have any ideas?

Reply:
You can set it to whatever here and then override it in ET+/W+ although that might make it harder to modify later on (more places to change)